src/main.c

```diff
@@ -144,12 +147,24 @@ typedef struct _VMSWITCH_PACKET_FRAGMENT {
     unsigned long RssHashValue;
 } VMSWITCH_PACKET_FRAGMENT, *PVMSWITCH_PACKET_FRAGMENT;
 
+typedef struct _RRAS_NDIS_WAN_PACKET_FRAGMENT {
+    char* RoutingDomainID;
+    char* RRASUserName;
+    unsigned long InterfacePreHashValue;
+} RRAS_NDIS_WAN_PACKET_FRAGMENT, *PRRAS_NDIS_WAN_PACKET_FRAGMENT;
+
 BOOLEAN CurrentPacketIsVMSwitchPacketFragment = FALSE;
 VMSWITCH_PACKET_FRAGMENT VMSwitchPacketFragment;
 
 const GUID NdisCapId = { // Microsoft-Windows-NDIS-PacketCapture {2ED6006E-4729-4609-B423-3EE7BCD678EF}
     0x2ed6006e, 0x4729, 0x4609, 0xb4, 0x23, 0x3e, 0xe7, 0xbc, 0xd6, 0x78, 0xef};
 
+BOOLEAN CurrentPacketIsRasNidsWanPacketFragment = FALSE;
+RRAS_NDIS_WAN_PACKET_FRAGMENT RasNdisWanPacketFragment;
```

Comment:
Should the "Ras" be "Rras"? Same question in CurrentPacketIsRasNidsWanPacketFragment